src/platform/packages/private/kbn-generate-csv/src/lib/search_cursor_pit.ts

```diff
@@ -47,13 +47,13 @@ export class SearchCursorPit extends SearchCursor {
           index: this.indexPatternTitle,
           keep_alive: scroll.duration(taskInstanceFields),
           ignore_unavailable: true,
-          // @ts-expect-error ignore_throttled is not in the type definition, but it is accepted by es
-          ignore_throttled: includeFrozen ? false : undefined, // "true" will cause deprecation warnings logged in ES
+          ...(includeFrozen ? { querystring: { ignore_throttled: false } } : {}), // "true" will cause deprecation warnings logged in ES
         },
         {
           signal: this.abortController.signal,
           requestTimeout: scroll.duration(taskInstanceFields),
           maxRetries: 0,
+          // @ts-expect-error not documented in the types. Is this still supported?
           maxConcurrentShardRequests,
```

Comment:
Elasticsearch 8.9+ supports it, but it was never added to the specification. I've opened https://github.com/elastic/elasticsearch-specification/pull/4142.